packages/status-js/src/client/community/community.ts

```diff
@@ -27,7 +27,7 @@ export class Community {
   public description!: CommunityDescription
   public chats: Map<string, Chat>
   #members: Map<string, Member>
-  public callback: ((description: CommunityDescription) => void) | undefined
+  #descriptionCallbacks: Set<(description: CommunityDescription) => void>
```

Comment:
rename to callbacks

Reply:
renamed at https://github.com/status-im/status-web/commit/e186ec7e7969c6451986a6d0b17f0d30106a0520